app/main.py

```diff
@@ -14,4 +14,4 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    return phrase.lower().count(letter.lower())
```

Comment:
Add empty line in the end of the file. It is a nit, but remember to do it always, so gitthub won't show these red circles and pycharm won't highlight it